File: src/app/gkill/api/req_res/get_lantana_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/app/gkill/api/message"
	"github.com/mt3hr/gkill/src/app/gkill/dao/reps"
)

type GetLantanaResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	LantanaHistories []*reps.Lantana `json:"lantana_histories"`
}

File: src/classes/api/req_res/get-lantana-response.ts
'use strict';

import { Lantana } from '@/classes/datas/lantana';
import { GkillAPIResponse } from '../gkill-api-response';


export class GetLantanaResponse extends GkillAPIResponse {


    lantana_histories: Array<Lantana>;

    constructor() {
        super()
        this.lantana_histories = new Array<Lantana>()
    }


}



